program.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,800&family=Raleway:wght@100;200;300;400&display=swap');
+
+.overlap {
+    position: relative;
+}
+
+.elements {
+    position: absolute;
+}
+
+
+.img {
+    float: right;
+    top: 180px;
+    right: 80px;
+    width: 873px;
+    height: 622px;
+}
+
+.panel {
+    padding-top: 50px;
+    padding-left: 100px;
+    padding-right: 80px;
+    padding-bottom: 100px;
+    margin-top: 170px;
+    width: 450px;
+    height: 450px;
+    top: 100px;
+    left: 80px;
+    background-color: rgb(204, 204, 204);
+}
+
+h1 {
+    font-family: 'Raleway', sans-serif;
+    font-size: 56px;
+    font-weight: 300;
+}
+
+.panel-text {
+    font-size: 18px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 200;
+    color:  rgb(77, 74, 74);
+    line-height: 32px;
+
+}
+
+.interest-form {
+    background-color: rgb(117,20,12);
+    color: white;
+    width: 150px;
+    height: 40px;
+    border-radius: 10px;
+    border: 0px;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+}
+
+.interest-form:hover {
+    background-color: gainsboro;
+}
+
+#the-program-h1 {
+    margin-top: 920px;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+    font-family: 'Raleway', sans-serif;
+    font-weight: 200;
+    font-size: 40px;
+
+}
+
+.panels{
+    margin-top: 70px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.panels > div {
+    border: 1px solid;
+    border-color: rgb(117,20,12);
+    box-shadow: 3px 3px 4px 0 rgba(107, 30, 35, 1);
+    height: 380px;
+    width: 340px;
+    margin: 20px;
+}
+
+.panels > div > h3 {
+    padding-top: 20px;
+    text-align: center;
+    font-size: 24px;
+    font-family: 'Raleway', sans-serif;   
+    font-weight: 300;
+}
+
+.panels > div > p {
+    margin: 15px;
+    font-family: 'Raleway', sans-serif;   
+    font-size: 16px;
+    font-weight: 400;
+}
+
+@media only screen and (max-width: 1200px) {
+    .img {
+        display: none;
+    }
+    .img {
+        float: right;
+        top: 1px;
+        right: 1px;
+        width: 1px;
+        height: 1px;
+    }
+    .overlap {
+        position: static;
+    }
+
+    .elements {
+        position: static;
+
+    }
+    
+    .panel {
```

Comment:
You don't have to redo all the attributes in the media selector, you only need to add the attributes you want to add/replace